src/server.c

```diff
@@ -2647,8 +2649,10 @@ int populateArgsStructure(struct redisCommandArg *args) {
     return count;
 }
 
-/* Recursively populate the command stracture. */
+/* Recursively populate the command structure. */
 void populateCommandStructure(struct redisCommand *c) {
+    c->fullname = sdsnew(c->fullname);
```

Comment:
i think we should have a `const char *declared_name` in the declarative part of `struct redisCommand` which comes from the structs in command.c and `sds fullname` in the run-time filled part

it's a bit dangerous that there's some amount of time when `fullname` is actually a `const char*`